### ISSUE TITLE
TODO: Add title

### ISSUE THREAD

**Issue by yassel998** [koalaman/shellcheck#1757]
#### For bugs
- Rule Id (if any, e.g. SC1000): SC2162, SC2162
- My shellcheck version (`shellcheck --version` or 'online'): online
- [x] I tried on shellcheck.net and verified that this is still a problem on the latest commit
- [ ] It's not reproducible on shellcheck.net, but I think that's because it's an OS, configuration or encoding issue

#### For new checks and feature suggestions
- [x] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [ ] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/bin/bash
read -p "enter a nbr" b
read -p "enter another nbr :" c
if [ "$b" -gt "$c" ]
then
echo "$b is greather that $c"
fi

```

#### Here's what shellcheck currently says:
```
Line 2	SC2162: read without -r will mangle backslashes.
Line 3	SC2162: read without -r will mangle backslashes.
```
#### Here's what I wanted or expected to see:

TODO: Describe expected/desired output

**Comment by Nightfirecat** [comment]
Can you provide some more detail for this issue? This sounds like an accurate case for https://github.com/koalaman/shellcheck/wiki/SC2162

**Comment by dimo414** [comment]
@yassel998 can you fix the issue title, and clarify what behavior you expected?
